### DISCUSSION TITLE
配置的 dumi targets 无效。

### DISCUSSION THREAD

**Discussion by hughfenghen** [umijs/dumi#2159] [Q&A]
截图是我的项目 `.dumirc.ts` 中的配置
![image](https://github.com/umijs/dumi/assets/3307051/315ce22f-50db-49a0-a671-1015de13d053)

但从devtools 中的调试，查看输出的代码的截图
![image](https://github.com/umijs/dumi/assets/3307051/c7247384-1574-493d-a897-03edd17a9976)

两个问题：
- dumi 将 async/await 语法编译成非常老版本的代码，不符合期望，项目的兼容目标是 chrome 102
- 编译了 node_modules 依赖模块的代码，与配置期望不符

请问是否有人碰到相似的问题，或是我的配置有什么问题吗？

**Comment by PeachScript** [comment]
`legacy` 是用来兼容 IE 的，优先级高于 `targets`，文档里有说明：https://d.umijs.org/config#legacy

MFSU 会预编译 `node_modules`，仅 dev 生效，想完全不编译可以关掉 MFSU

**Reply by hughfenghen** [reply]
你好，感谢回复 ：）

我按照建议修改配置
![image](https://github.com/umijs/dumi/assets/3307051/fb8e9633-c13c-473c-9a9c-28a0aac6c3f5)

build dev 打包仍然会编译 async/await 语法，这是 dev 模式的截图
![image](https://github.com/umijs/dumi/assets/3307051/c9cf66a6-9877-41e0-8676-d32e30f821a2)


### 其它信息

- 重新构建，node_module/.cache 目录下，msfu 目录已经没了
- 项目是 monorepo 结构，通过 alias 引入的依赖
![image](https://github.com/umijs/dumi/assets/3307051/29912184-8603-447b-8c9c-c88c2d80a812)


## 版本信息
```
"@umijs/plugins": "^4.1.2",
"dumi": "^2.2.13",
```

**Reply by PeachScript** [reply]
@Jinbao1001 看看这个和你在查的是不是同一个问题

**Reply by Jinbao1001** [reply]
应该是 umi 的 bug, 目前默认的 + 配置的 targets 都不生效, babel 会无脑转低版本浏览器, 我近期会修复下. @hughfenghen

**Reply by hughfenghen** [reply]
太棒了，[给大佬递茶]
